session_0/word_processing.rb

```diff
@@ -18,7 +30,16 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+  i = 0
+  l = words.length
+  output = Array.new(0)
+  while i < l do
```

Comment:
Checkout the select construct
https://www.rubyguides.com/2019/04/ruby-select-method/